aperturedb/PyTorchDataset.py

```diff
@@ -164,6 +165,6 @@ def get_batch(self, index):
             else:
                 self.batch_labels = ["none" for l in range(len(b))]
         except:
-            print("Query error:")
-            print(self.db.get_last_response_str())
+            logger.error("Query error:")
+            logger.error(self.db.get_last_response_str())
```

Comment:
Can we do something like this here?:

```suggestion
            logger.error("Query error:", self.db.get_last_response_str())
```